components/homme/src/share/cxx/HommexxEnums.hpp

```diff
@@ -106,25 +106,14 @@ enum class UpdateType {
 
 enum class TimeStepType {
   // Explicit
-  LF              =  0, // LeapFrog
-  RK2             =  1,
-  IMEX_KG254_EX   =  4, // Explicit table from IMEX_KG254
-  ULLRICH_RK35    =  5,
+  ttype5          =  5,
```

Comment:
I think I might be in the minority here, but I'll say it anyways.

I think that in C++ we should take advantage of enums, and not use names like "ttype5", or "q_alg_10", or "ftype2". If we prefer ints, then let's use ints. Enum's are a way to convey meaning like a string, while keeping the underlying simplicity of an int.

That said, I understand that ttype being an int in the namelists is an argument for keeping the status quo, so I'm not going to push on this.